### PR TITLE
Metal press rendersync

### PR DESCRIPTION
**Pull request by MalkContent**
gets rid of a lot of magic numbers in MetalPressRenderer and MetalPressTileEntity and centralizes them in MetalPressRenderer to allow for easier manipulation of the machines operating speed/behavior

added code to automatically speed up the transport and press speeds for processes with maxTicks smaller than the standard operating speed allow by default

removed usage of partialTicks because of at seemingly randomly timed jaggy animation of the items on the conveyor belt, more noticably at faster processing speeds
sadly could not find a solution to make it use partialTicks properly :/

### CODE REVIEW

File: src/main/java/blusunrize/immersiveengineering/client/render/tile/MetalPressRenderer.java [hunk 1]
```diff
@@ -59,30 +63,33 @@ public void render(MetalPressTileEntity te, double x, double y, double z, float
 		GlStateManager.translated(x+.5, y+.5, z+.5);
 		float piston = 0;
 		float[] shift = new float[te.processQueue.size()];
+		System.out.println(partialTicks);
```
Comment (SkySom) [inline src/main/java/blusunrize/immersiveengineering/client/render/tile/MetalPressRenderer.java]: println needs to be removed.

**Review by SkySom — CHANGES_REQUESTED** [review]

File: src/main/java/blusunrize/immersiveengineering/client/render/tile/MetalPressRenderer.java (hunk 1 above)
Comment (MalkContent) [reply src/main/java/blusunrize/immersiveengineering/client/render/tile/MetalPressRenderer.java]: ty, good catch

**Review by SkySom — APPROVED** [review]

**Comment by BluSunrize** [comment]
This should be obsolete due to #4272, correct?

**Comment by MalkContent** [comment]
correct, it's included in #4272

**Pull request closed without merging**